Server/Ebenezer/User.cpp

```diff
@@ -11515,6 +11529,11 @@ BOOL CUser::CheckEventLogic(EVENT_DATA* pEventData)
 					bExact = TRUE;
 				break;
 
+			case LOGIC_CHECK_EMPTY_SLOT:
+				if(GetNumberOfEmptySlots() >= pLE->m_LogicElseInt[0])
```

Comment:
Select this and run a quick formatter pass (press CTRL+K, then press F) over it.